pennylane_qiskit/qiskit_device.py

```diff
@@ -326,15 +343,26 @@ def _get_state(self, result):
         # reverse qubit order to match PennyLane convention
         return state.reshape([2] * self.num_wires).T.flatten()
 
-    def generate_samples(self):
+    def generate_samples(self, circuit=None):
```

Comment:
This method used to inherit its docstring from `QubitDevice` (now takes the `circuit` kwarg).